optaplanner-quarkus-integration/optaplanner-quarkus-jsonb/pom.xml

```
@@ -13,9 +13,29 @@
   <name>OptaPlanner Quarkus JSON-B</name>
   <packaging>pom</packaging>
 
-  <modules>
-    <module>runtime</module>
-    <module>deployment</module>
-    <module>integration-test</module>
-  </modules>
+  <profiles>
+    <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <modules>
+        <module>runtime</module>
+        <module>deployment</module>
+        <module>integration-test</module>
+      </modules>
+    </profile>
+    <profile>
+      <id>prod</id>
+      <activation>
+        <property>
+          <name>prod</name>
```

i would use the same property we use in other assembllies
```suggestion
          <name>productized</name>
```